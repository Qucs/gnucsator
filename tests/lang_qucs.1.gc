#>.load d_probe.so
#>.load bm_value.so
#>.load bm_wrapper.so
#>.load cmd_wrapper.so
#>.load bm_trivial.so

R:R1 _net0 _net1 R="50 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
Vdc:V1 _net1 _net0 U="1.3" Temp="2"
VProbe:Pr1 _net0 _net1

#>.print tran v(nodes) v(V1) r(R1)
#>.list
.TR:mySim Start="0" Stop="1 ms" Points="5" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
