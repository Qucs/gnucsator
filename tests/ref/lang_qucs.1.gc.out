;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
dev_comment: already installed, replacing
stashing as dev_comment:0
module tst (1,2);
resistor #() r (.p(1),.n(2));
endmodule // tst

.Def:tst 1 2 
  resistor:r 1 2  
.Def:End
